README.md

```diff
@@ -1,162 +1,74 @@
-
-[![Build Status](https://secure.travis-ci.org/mozilla/Addon-Tests.png?branch=master)](http://travis-ci.org/mozilla/Addon-Tests/)
+[![license](https://img.shields.io/badge/license-MPL%202.0-blue.svg)](https://github.com/mozilla/Addon-Tests/blob/master/LICENSE)
+[![travis](https://img.shields.io/travis/mozilla/Addon-Tests.svg?label=travis)](http://travis-ci.org/mozilla/Addon-Tests/)
+[![dev](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/amo.dev.svg?label=dev)](https://webqa-ci.mozilla.com/view/AMO/job/amo.dev/)
+[![stage](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/amo.stage.svg?label=stage)](https://webqa-ci.mozilla.com/view/AMO/job/amo.stage/)
+[![prod](https://img.shields.io/jenkins/s/https/webqa-ci.mozilla.com/amo.prod.svg?label=prod)](https://webqa-ci.mozilla.com/view/AMO/job/amo.prod/)
 
 Getting involved as a contributor
 ------------------------------------------
 
-We love working with contributors to fill out the Selenium test coverage for Addon-Tests, but it does require a few skills.   You will need to know some Python, some Selenium and you will need some basic familiarity with Github.
+We love working with contributors to fill out the Selenium test coverage for Addon-Tests, but it does require a few skills. By contributing to our test suite you will have an opportunity to learn and/or improve your skills with Python, Selenium WebDriver, GitHub, Virtualenv, the Page Object Model, and more.
 
-If you know some Python, it's worth having a look at the Selenium framework to understand the basic concepts of browser based testing and especially page objects. Our suite uses [Selenium WebDriver][webdriver].
+For some resources for learning about these technologies, take a look at our documentation on [Running Web QA automated tests][runningtests].
 
-If you need to brush up on programming but are eager to start contributing immediately, please consider helping us find bugs in Mozilla [Firefox][firefox] or find bugs in the Mozilla web-sites tested by the [WebQA][webqa] team.
+The following contributors have submitted pull requests to Addon-Tests:
 
-To brush up on Python skills before engaging with us, [Dive Into Python][dive] is an excellent resource.  MIT also has [lecture notes on Python][mit] available through their open courseware.The programming concepts you will need to know include functions, working with classes, and some object oriented programming basics. 
+https://github.com/mozilla/Addon-Tests/contributors
 
-[mit]: http://ocw.mit.edu/courses/electrical-engineering-and-computer-science/6-189-a-gentle-introduction-to-programming-using-python-january-iap-2011/
-[dive]: http://www.diveintopython.net/toc/index.html
-[webqa]: http://quality.mozilla.org/teams/web-qa/
-[firefox]: http://quality.mozilla.org/teams/desktop-firefox/
 [webdriver]: http://seleniumhq.org/docs/03_webdriver.html
+[runningtests]: https://developer.mozilla.org/en-US/docs/Mozilla/QA/Running_Web_QA_automated_tests
```

Comment:
I think it would be better to move the links for webdriver & runningtests above the contributors blob to line 18, or below into the questions area. They are free floating right now which removes context.